sycl/include/CL/sycl/ONEAPI/matrix/matrix-amx.hpp

```diff
@@ -179,8 +180,8 @@ template <typename Group, typename T, size_t NumRows, size_t NumCols,
           matrix_layout Layout>
 struct joint_matrix<
     Group, T, NumRows, NumCols, Layout,
-    typename std::enable_if<(NumRows <= tile_size) &&
```

Comment:
Why are these changes part of this PR?  My understanding was this PR was for namespace and file-name changes specifically.

Reply:
Eh, @alexbatashev suggested that we use enable_if_t in his comments. I am ok if we change it back or keep code there.

Reply:
I think that should be handled in a separate PR.  While I agree that is an improvement, it is not part of fixing the namespace and file-name issues.

Reply:
OK, I will move it into another new PR to handle it.

Reply:
I'm ok with committing as a separate PR as well